src/main/java/seedu/address/logic/commands/AssignCommand.java

```diff
@@ -75,15 +87,29 @@ public CommandResult execute(Model model) throws CommandException {
             personToAssign = assignWithKeyword(model);
         }
 
-        Person assignedPerson = createPersonWithRole(personToAssign, personWithRoleDescriptor);
+        Person assignedPerson;
+        if (weddingIndices != null) {
```

Comment:
Maybe can add the checks for duplicated role? Also, maybe can add checks for duplicated wedding jobs added, to prevent such cases?